plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptionsVariationSelectorAttributeOptions.php

```diff
@@ -150,13 +150,15 @@ protected function render_pills( $attributes, $content, $block ) {
 		$pills = '';
 		foreach ( $attribute_terms as $attribute_term ) {
 			$pills .= sprintf(
-				'<div %s>%s</div>',
+				'<li %s>%s</li>',
```

Comment:
To match the markup from the editor. I'm unsure which of the two is more semantically correct, to be honest